src/__tests__/testcases/overrides/index.ts

```diff
@@ -14,7 +14,7 @@ export default class Foo {
   method(c: C): D;
   method(e: E): F;
 
-  method(c: C | D): D | F {
-    throw c;
+  method(ce: C | E): D | F {
```

Comment:
oh, interesting. Even though tests are excluded from the main tsconfig, this should have still given a local error, no?

Reply:
`D` and `E` are empty interfaces so they are *structurally* identical, which is fine for the TypeScript compiler. 😄

Reply:
Haha, you are absolutely right. Reminds me how I miss nominal typing :-D